.travis.yml

```diff
@@ -3,22 +3,35 @@ cache: bundler
 script: bundle exec rake
 rvm:
   - 2.1.10
-  - 2.2.6
-  - 2.4.1
+  - 2.2.10
+  - 2.4.4
+  - 2.5.1
 gemfile:
   - gemfiles/Gemfile-rails.4.1.x
-  - gemfiles/Gemfile-rails.head
+  - gemfiles/Gemfile-rails.5.1.x
   - Gemfile
+  - gemfiles/Gemfile-rails.head
 matrix:
   exclude:
-    - rvm: 2.2.6
-      gemfile: gemfiles/Gemfile-rails.4.1.x
-    - rvm: 2.4.1
+    - rvm: 2.1.10
       gemfile: gemfiles/Gemfile-rails.4.1.x
     - rvm: 2.1.10
-      gemfile: gemfiles/Gemfile-rails.head
+      gemfile: gemfiles/Gemfile-rails.5.1.x
     - rvm: 2.1.10
       gemfile: Gemfile
+    - rvm: 2.1.10
+      gemfile: gemfiles/Gemfile-rails.head
+    - rvm: 2.2.10
+      gemfile: gemfiles/Gemfile-rails.4.1.x
```

Comment:
Could we drop these Rails old versions? 🤔

Reply:
Maybe, but I think that's a matter for other PR. Since none of the tests broke I think we can leave as it is for now.